database/index.js

```diff
@@ -36,16 +35,23 @@ let headerSchema = {
 
 const HeaderModel = mongoose.model('headerData', new Schema (headerSchema));
 let SeedData = [];
-// for (let i = 0; i < 100; i++) {
-//   let generatedData = data.objectCreation(i);
-//   let currentModel = new HeaderModel(generatedData);
-//   SeedData.push(currentModel.save());
-// .catch((err) => {
-//   throw new Error(err);
-// });
-// SeedData.push(currentModel.update({upsert: true}));
-// }
-// Promise.all(SeedData);
+
+HeaderModel.find({})
+  .then((requestData) => {
+    if (requestData.length < 100) {
+      for (let i = 0; i < 100; i++) {
+        let generatedData = data.objectCreation(i);
```

Comment:
I forgot mongoose for the most part so can you tell me what is `data.objectCreation(i)` here? Where is `data` coming from and `objectCreation` as well?

Reply:
data is the database file that I imported and objectCreation is a function from that file. I have updated the naming.